.github/workflows/release.yml

```diff
@@ -3,25 +3,16 @@ name: Release on PyPI
 on:
   push:
     tags:
-    - v*
+      - v*
 
 jobs:
-  deploy:
+  publish-package:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.11'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install setuptools wheel twine
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python setup.py sdist bdist_wheel
-        twine upload dist/*
+      - name: Publish PyPi package
```

Comment:
A little note: we don't actually publish this on PyPI. But I guess this can stay here for now.